platform/ui/src/components/customForm/HotkeyField.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { utils } from '@ohif/core';
+
+const { hotkeyRecord } = utils;
+
+/**
+ * Take the pressed key array and return the readable string for the keys
+ *
+ * @param {Array} [keys=[]]
```

Comment:
My nitty-gritty self wants to say that it's nice to have @returns specified even when it's just void, my other self also forgets to add returns. 😆

Reply:
Added :)